kahuna/public/js/components/gr-image-usage/gr-image-usage.html

```diff
@@ -0,0 +1,35 @@
+<div ng:show="ctrl.usage.pending || ctrl.usage.published">
```

Comment:
In future I suggest keeping UI changes separate from back-end work (e.g. on a separate branch based on this one), in case we want to merge/release/revert them independently. Unless there was a reason to do this here?